globals.go

```diff
@@ -213,9 +213,29 @@ const (
 	// app server should log to stdout.
 	standardOutputLoggingOverride = "STDOUT"
 
-	DefaultTaskActivator   = ""
-	StepbackTaskActivator  = "stepback"
+	// DefaultTaskActivator is a deprecated legacy activator that used
```

Comment:
should we make a ticket so we don't forget to get rid of this at some point? Presumably we can drop it a year from now

Reply:
Created https://jira.mongodb.org/browse/EVG-20869